Bender.yml

```diff
@@ -18,16 +22,17 @@ dependencies:
   icache_mp_128_pf:       { git: "https://github.com/pulp-platform/icache_mp_128_pf.git", rev: "6f2e54102001230db9c82432bf9e011842419a48" }
   icache_private:         { git: "https://github.com/AlSaqr-platform/icache_private.git", rev: "84436443d3a2dbad0b94a85c2c5cc8915632fd26" }
   cluster_peripherals:    { git: "https://github.com/pulp-platform/cluster_peripherals.git", version: 2.1.0 }
-  fpu_interco:            { git: "https://github.com/AlSaqr-platform/fpu_interco.git", rev: "35b582ee6683b65465d34b44ef289a9832dfd516" }
-  axi:                    { git: "https://github.com/pulp-platform/axi.git", rev: "a25b415958e83e232096aa3863da56887abd5991" } 
+  fpu_interco:            { git: "https://github.com/pulp-platform/fpu_interco.git", rev: "2f4a1b594bc4ff3b68d55e569fbe912ac121924f" }
```

Comment:
are we sure about this? has this been tested?

Reply:
Updated now.

Reply:
Needs some tests at least.